test/lib_edit_address/factory/edit_address_widget.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/edit_address.dart';
+import 'package:widget_toolkit/language_picker.dart';
+import 'package:widget_toolkit/src/lib_edit_address/blocs/edit_address_bloc.dart';
+
+import '../mock/edit_address_mock.dart';
+import '../repository/search_country_repository_mock.dart';
+import '../service/edit_address_fields_service_mock.dart';
+import '../service/search_country_service_mock.dart';
+import '../utils/edit_address_error_mapper_util.dart';
+
+/// Change the parameters according the the needs of the test
+Widget editAddressWidget({
+  AddressModel? onAddressSet,
+  bool? isLoading,
+  ErrorModel? errors,
+  String? street,
+  String? city,
```

Comment:
Name widgets with capital-camel-case and add Factory sufix

Reply:
I renamed the method to createEditAddressWidget and moved it to edit_address_page_factory.dart file